packages/eslint-config/index.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
   rules: {
     eqeqeq: ["error", "smart"],
     "no-console": ["error"],
+    "sonarjs/cognitive-complexity": ["error", 16]
```

Comment:
Please use the suggestions shared by the team to decrease the complexity. The following change will affect the whole codebase